creme/settings.py

```diff
@@ -888,6 +888,7 @@
     ('creme.billing',       'billing/js/tests/billing.js'),
     ('creme.billing',       'billing/js/tests/billing-actions.js'),
     ('creme.billing',       'billing/js/tests/billing-listview.js'),
+    ('creme.commercial',    'commercial/js/tests/commercial-assets.js'),
```

Comment:
"asset" is not very relevant ; there is asset-score, there is charm-score. It's all about "score", not "asset".

Reply:
You are right, `commercial-score.js` is a better name